src/Microsoft.DotNet.Interactive/KernelExtensions.cs

```diff
@@ -208,6 +209,194 @@ public static T UseLogMagicCommand<T>(this T kernel)
         return kernel;
     }
 
+    public static T UseSet<T>(this T kernel) where T : Kernel
```

Comment:
```suggestion
    public static T UseSetMagicCommand<T>(this T kernel) where T : Kernel
```

Reply:
This naming would be more informative and consistent but on second thought, I think this functionality should just be part of `UseValueSharing`. It's the same core functionality and there's no use case for enabling one and not the other.

Reply:
Just moved this as part of turning on ValueSharing, refactored some code to become reused (sendValue for example)